_data/domains.yml

```diff
@@ -460,6 +460,13 @@ websites:
       tfa: No
       lang: de
 
+    - name: UK2
+      url: https://www.uk2.net/
+      twitter: UK2
+      facebook: uk2.net
+      img: 
```

Comment:
could you add the filename `uk2.png` for the `img` tag?

also, rename the img to `uk2.png`? we try to only have the entry `name` in the `img` filename

it should look like this: `img: uk2.png`